Sources/Client/Client/Client.swift

```diff
@@ -58,36 +58,35 @@ public final class Client {
     public var connection: Connection { webSocket.connection }
     
     /// A WebSocket connection callback. This should only be used when you only use the Low-Level Client.
-    public var onConnect: Client.OnConnect = { _ in } {
-        didSet { webSocket.onConnect = setupWebSocketOnConnect }
-    }
+    public var onConnect: Client.OnConnect = { _ in }
     
     /// Saved onConnect for a completion block in `connect()`.
     var savedOnConnect: Client.OnConnect?
     
-    /// A WebSocket events callback. This should only be used when you only use the Low-Level Client.
-    public var onEvent: Client.OnEvent = { _ in } {
-        didSet { webSocket.onEvent = setupWebSocketOnEvent }
-    }
-    
     lazy var urlSession = URLSession(configuration: .default)
     lazy var urlSessionTaskDelegate = ClientURLSessionTaskDelegate() // swiftlint:disable:this weak_delegate
     let callbackQueue: DispatchQueue?
-    private let uuid = UUID()
     
     /// A log manager.
     public let logger: ClientLogger?
     public let logOptions: ClientLogger.Options
     
-    /// An observable user. This should only be used when you only use the Low-Level Client.
-    public var onUserUpdate: OnUpdate<User>?
+    private(set) lazy var userUpdateHandlingQueue = DispatchQueue(label: "io.getstream.StreamChatClient.userUpdate", qos: .userInteractive)
```

Comment:
We no need this queue, mostly updates will be from websockets. 
Can you use websocket queue?

Reply:
We _do_ need this queue, `webSocket.webSocket.callbackQueue` is not available here and user updates are not coming from `WebSocket`, they come from `User`.

This is not possible since onUserUpdate is called from User:
```swift
let unreadCountAtomic = Atomic<UnreadCount>(.noUnread) { _, _ in Client.shared.onUserUpdate(User.current) }
```
an alternative would be:
```swift
let unreadCountAtomic = Atomic<UnreadCount>(.noUnread) { _, _ in Client.shared.userAtomic.set(User.current) }
```
which I fear might have side effects.